Examples/ChemDrawOAuthExample/src/App.test.tsx

```diff
@@ -1,7 +1,7 @@
 //
 // App.test.tsx
 //
-// Copyright © 2019 PerkinElmer, Inc. All rights reserved.
```

Comment:
Not sure if we should do this. Doing this means that will need to do this every year.

But doing this means we are actively maintaining this public repo, which makes us look good.

FYI. Some discussions on the "year" in the copyright:
https://www.copyrightlaws.com/copyright-symbol-notice-year/

```
WHICH YEAR SHOULD YOU USE IN YOUR COPYRIGHT NOTICE?

The general rule is that the year to include in a copyright notice is the year
of first publication of the work. First publication is when the work is made
available to the public without restriction. This includes selling a book or
leaving free copies of it in a public place or posting content on your blog.
Publication does not include, for example, circulating copies internally to
co-workers or sending copies of a book to a publisher.

For compilations or derivative works which incorporate previously published
content, the year of first publication of the compilation or derivative work is
sufficient.
```

```
IF I BEGAN WRITING A BOOK IN 2011, FINISHED IT IN 2012, AND PUBLISHED
IT IN 2020, WHAT IS THE PROPER COPYRIGHT NOTICE YEAR?

Use the date of first publication, which in this case is 2020.
```

Reply:
`But doing this means we are actively maintaining this public repo, which makes us look good` This is what suggested by PO https://jira-ext.perkinelmer.com/browse/DSK-23052
But I agree with what you have suggested